WalletWasabi/Wallets/Wallet.cs

```diff
@@ -28,7 +28,7 @@ namespace WalletWasabi.Wallets;
 public class Wallet : BackgroundService, IWallet
 {
 	private volatile WalletState _state;
-	private static int WalletCount;
+	private static int WalletCount = 1;
```

Comment:
Why 1? Then the first ID is going to be 2

Reply:
Argh... I wrongly updated the ++Id (operator precedence) without even thinking about it... Sorry.